CHANGELOG.next.asciidoc

```diff
@@ -191,6 +191,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - elasticsearch output now supports `idle_connection_timeout`. {issue}35616[35615] {pull}36843[36843]
 - Enable early event encoding in the Elasticsearch output, improving cpu and memory use {pull}38572[38572]
 - The environment variable `BEATS_ADD_CLOUD_METADATA_PROVIDERS` overrides configured/default `add_cloud_metadata` providers {pull}38669[38669]
+- Kafka output allows dynamic topic in `topic` field {pull}40415[40415]
```

Comment:
I believe it would be good to include that this is only under Elastic-Agent because standalone Beats have always supported this.
```suggestion
- When running under Elastic-Agent Kafka output allows dynamic topic in `topic` field {pull}40415[40415]
```